sdcm/utils/common.py

```diff
@@ -297,6 +297,21 @@ def convert_to_date(date_str):
     return results
 
 
+def search_parallel_timelines_report_urls(test_id):
```

Comment:
Why do we need it in `common.py`?

Reply:
I placed this function in common.py because we have there is a similar function for S3 scanning. So I thought this module suited well. Could you please advise a more appropriate place for it?

Reply:
I see. `S3Storage` and related functions are good candidates to be extracted from `common.py`, I think. But this is outside of the scope of this PR. @bentsi ?

Ok, let's stay here but rename this function to `list_..._by_test_id` to be consistent with naming.

Reply:
Renamed